source/read/count.txt

```diff
@@ -0,0 +1,138 @@
+.. _kotlin-sync-count:
+
+===============
+Count Documents
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: number, amount, estimation, code example
+
+Overview
+---------
+
+In this guide, you can learn how to retrieve accurate and estimated counts of the
+number of documents in a collection.
+
+.. _kotlin-sync-accurate-count:
+
+Retrieve an Accurate Count
+--------------------------
+
+Use the ``countDocuments()`` method to count the number of documents that are in a
+collection. To count the number of documents that match specified search
+critera, pass a query filter to the ``countDocuments()`` method.
+
+.. To learn more about specifying a query, see :ref:`kotlin-sync-specify-query`.
+
+Count All Documents
+~~~~~~~~~~~~~~~~~~~
+
+To return a count of all documents in the collection, call the ``countDocuments()`` method
+with no arguments, as shown in the following example:
+
+.. code-block:: kotlin
+
+   collection.countDocuments()
+
+Count Specific Documents
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+To return a count of documents that match specific search criteria, specify your query
+in the ``countDocuments()`` method, as shown in the following example:
+
+.. code-block:: kotlin
+   
+   collection.countDocuments(eq("author", "Leslie"))
+
+Customize Count Behavior
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``countDocuments()`` method accepts optional parameters in the form of a
+``CountOptions`` object, which represents options you can use to configure the count
```

Comment:
Q: how do you set these options - by specifying fields of a `CountOptions` object? Might be good to clarify that

Reply:
Can add some clarifying wording.